src/jdk.compiler/share/classes/com/sun/tools/javac/launcher/MemoryClassLoader.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2023, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Hello Christian, this should be `2023, 2024, `

Reply:
Good catch, will fix it.